src/symbol/glyph_source.js

```diff
@@ -5,6 +5,9 @@ const verticalizePunctuation = require('../util/verticalize_punctuation');
 const Glyphs = require('../util/glyphs');
 const GlyphAtlas = require('../symbol/glyph_atlas');
 const Protobuf = require('pbf');
+const TinySDF = require('@mapbox/tiny-sdf');
+const isChar = require('../util/is_char_in_unicode_block');
+/* eslint-disable new-cap */
```

Comment:
Hmm, I mean, that line is supposed to disable the new-cap rule for the current file (which doesn't have any occurences of `new lowercaseThing()`. So should this move to specific lines where it is used?